ironfish/src/utils/currency.ts

```diff
@@ -69,6 +69,20 @@ export class CurrencyUtils {
 
     return ore
   }
+
+  static renderBalance(amount: bigint | string, assetIdentifier?: string): string {
```

Comment:
The code above should be converted to use an optional assetIdentifier instead of adding this function. `Iron` and `Ore` refer to denominations rather than assets in the above context. It just defaults to the IRON ticket.

Reply:
Yep, works for me